presenter-in-iframe/index.html

```diff
@@ -0,0 +1,147 @@
+            <button
+              onclick="getCostQueryForItem('${message.data.type}', '${message.data.id}')"
+            >
+              <strong> Get Cost Query For id ${message.data.id} </strong>
+            </button>
+          </div>
+        </div>
+        <div style="flex: 0 1 auto">
+          <h4>Event log:</h4>
+        </div>
+        <div style="width: 100%; overflow-y: scroll; flex: 1">
+          <ol id="event_log" reversed></ol>
+        </div>
+      </div>
+      <iframe
+        style="margin: auto; width: 100%; height: 100%"
+        id="thisIframe"
+        src="<PRESENTER URL GO HERE>"
+      >
+      </iframe>
+    </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.19.1/moment.js"></script>
+    <script>
+      function postToIframe(message) {
+        const iframe = document.getElementById("thisIframe");
+        console.log("postToIframe", message);
+        iframe.contentWindow.postMessage(message, "*");
+      }
+      function getCostQuery() {
+        const message = {
+          event: "getCostQuery",
+        };
+        postToIframe(message);
+      }
+
+      function getCostQueryForItem(type, id) {
+        const message = {
+          event: "getCostQuery",
+          data: {
+            items: [
+              {
+                type,
+                id,
+              },
+            ],
+            queryId: `query_${id}`,
+          },
+        };
+        postToIframe(message);
+      }
+
+      function computeCostUsingServer() {
+        const message = {
+          event: "computeCostUsingServer",
+        };
+        postToIframe(message);
+      }
+      function getProjectData() {
+        const message = {
+          event: "getProjectData",
+        };
+        postToIframe(message);
+      }
+
+      function focusOn(type, id) {
+        const message = {
+          event: "zoomOn",
+          data: {
+            type,
+            id,
+          },
+        };
+        postToIframe(message);
+      }
+      function onMessage(message) {
+        if (message.data.type) {
+          const focusOn = document.getElementById("focusOn");
+          focusOn.innerHTML = `
+          <div style="width: 100%; flex: 0 1 auto" id="focusOn">
+            <button onclick="focusOn('${message.data.type}', '${message.data.id}')">
+              <strong> Focus on ${message.data.type} with id ${message.data.id} </strong>
+            </button>
+          </div>
+          <div style="width: 100%; flex: 0 1 auto">
+            <button onclick="getCostQueryForItem('${message.data.type}', '${message.data.id}')">
+              <strong> Get Cost Query For id ${message.data.id} </strong>
+            </button>
+          </div>
+            `;
+        }
+        addTextNode(JSON.stringify(message.data, null, 2));
+      }
+      function addTextNode(text) {
+        var newDiv = document.createElement("li");
+        newDiv.innerHTML = `[${moment().format(
```

Comment:
Can you just use `(new Date()).toLocaleTimeString()` here? Then you don't need moment.js.